src/Microsoft.AspNet.Mvc.Core/ApplicationModels/IApiExploringModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Microsoft.AspNet.Mvc.ApplicationModels
+{
+    public interface IApiExploringModel
```

Comment:
Why the verb in the name? I'd prefer `IApiExplorerModel`. Were you looking to avoid confusion between `ApiExplorerModel` and `IApiExplorerModel`?